### ISSUE TITLE
Automatically strip indentation from markdown content

### ISSUE THREAD

**Issue by falkoschindler** [zauberzeug/nicegui#617]
While working on the documentation (#439) I created a little helper function to remove common indentation from markdown code:
https://github.com/zauberzeug/nicegui/blob/c4d1e800e8f1d7b60a1da8108a5fca108bbbc418/website/documentation_tools.py#L16-L24

This allows writing something like

```py
with ui.card():
    ui.markdown('''
        ### Hello world
        This is a test.
    ''')
```

which would otherwise be rendered as code block due to the indentation.

Maybe this would be a useful feature for `ui.markdown`. I can't think of a use case where the indentation of the first (non-whitespace) line is important for the markdown interpretation. So we could strip it by default.

**Comment by rodja** [comment]
I think this it is very reasonable to performed on the input of `ui.markdown` function. Anybody disagrees? If necessary we could introduce a parameter "no_indentation_adjustment" or similar...

**Comment by falkoschindler** [comment]
Ok, it's implemented, tested and documented in [be6d5cd](https://github.com/zauberzeug/nicegui/commit/be6d5cd0d23d413afd2ec6012024e18b8b2f28f0) and [3ceef7c](https://github.com/zauberzeug/nicegui/commit/3ceef7cc87aff1996fdb9c859acf9c04db5be08c),.
